roofit/roofitcore/CMakeLists.txt

```diff
@@ -456,6 +456,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
     Matrix
     Tree
     Minuit
```

Comment:
Is there a Minuit dependency? @guitargeek ?
By now, this might be behind interfaces.